src/components/SignUpSuccessForm/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const SignUpSuccessForm = () => {
+  return (
+    <section className="sign-up-success-form">
+      <header className="sign-up-success-form__header">Sample Picture</header>
```

Comment:
For `header` tag, I think it would be better to use it with `h1, h2, h3, ..`. Let's just use `div` for this. What do you think?
```suggestion
      <div className="sign-up-success-form__image">Sample Picture</header>
```

Reply:
I agree with you because ``header`` tag should be kind of header for an article but for this one just using ``div`` tag to represent the image inside.